test/functional/mempool_persist.py

```diff
@@ -85,5 +91,29 @@ def run_test(self):
         self.nodes.append(self.start_node(0, self.options.tmpdir))
         wait_until(lambda: len(self.nodes[0].getrawmempool()) == 5)
 
+        regtestdir0 = os.path.join(self.options.tmpdir, 'node0', 'regtest')
+        mempooldat0 = os.path.join(regtestdir0, 'mempool.dat')
+        regtestdir1 = os.path.join(self.options.tmpdir, 'node1', 'regtest')
+        mempooldat1 = os.path.join(regtestdir1, 'mempool.dat')
+        self.log.debug("Remove the mempool.dat file. Verify that dumpmempool to disk via RPC re-creates it")
```

Comment:
Supernit: I like logging in the test script to be 'info' level, so when running the test locally, progress can be seen on the console by default.

Reply:
All the other logs in the test are debug. Either we change them all or I don't think it makes sense for me to just change mine.